Community.ManagementPackCatalog.UI/WpfElements/MarkdownViewer.xaml.cs

```diff
@@ -0,0 +1,71 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="MarkdownViewer.xaml.cs">
+// This is part of the Community Management Pack Catalog and licensed GPL v3; See https://github.com/mpcatalog/scom-community-catalog-ui/blob/master/LICENSE.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Community.ManagementPackCatalog.UI.WpfElements
+{
+    using System.Windows.Controls;
+    using static Community.ManagementPackCatalog.UI.LogManager;
+
+    /// <summary>
+    /// Interaction logic for MarkdownViewer.xaml
+    /// </summary>
+    public partial class MarkdownViewer : UserControl
+    {
+        private string markdownText;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MarkdownViewer"/> class
+        /// </summary>
+        public MarkdownViewer()
+        {
+            InitializeComponent();
+            MarkdownBrowserView.Navigating += MarkdownBrowserView_Navigating;
+        }
+
+        /// <summary>
+        /// Clears out the control when there is no text to display.
+        /// </summary>
+        public void ClearMarkdown()
+        {
+            MarkdownBrowserView.NavigateToString("<br/>");
+        }
+
+        /// <summary>
+        /// Render the provided markdown text in the window.
+        /// </summary>
+        /// <param name="markdownText">MarkDown text to display</param>
+        public void DisplayMarkdown(string markdownText)
+        {
+            this.markdownText = markdownText;
+            string htmlToRender = Markdig.Markdown.ToHtml(markdownText);
```

Comment:
Markdig is awesome :)

Reply:
I added an inline CSS to modernize the look and have had great results with it.

Reply:
Yeah, we use it for our internal docs platform and it's awesome. We've even built out a few additional extensions for it :)